data/data.go

```diff
@@ -21,17 +21,16 @@ func GetCluster(
 	c InstalledData,
 	i ClusterID,
 	v AvailableComponentVersion,
-	secretGetterCreator KubeSecretGetterCreator,
 ) (models.Cluster, error) {
 
 	// Populate cluster object with installed components
 	cluster, err := GetInstalled(c)
 	if err != nil {
 		return models.Cluster{}, err
 	}
-	err = AddUpdateData(&cluster, v, secretGetterCreator)
+	err = AddUpdateData(&cluster, v)
 	if err != nil {
-		return models.Cluster{}, err
+		log.Println("unable to decorate cluster data with available updates data")
```

Comment:
1. I don't want the failure of determining whether or not a newer component versions are available (what `AddUpdateData` does) to hijack the entire cluster data gathering process. I would prefer to simply return the cluster data without any update data decorations, from which the user would infer that available update data is undetermined.
2. will fix